client/src/components/Endpoints/getIcon.tsx

```diff
@@ -38,17 +53,17 @@ const getIcon = (props) => {
       icon = <GPTIcon size={size * 0.7} />;
       bg = 'linear-gradient(0.375turn, #61bde2, #4389d0)';
       name = chatGptLabel || 'ChatGPT';
-    } else if (endpoint === 'openAI' || (endpoint === 'gptPlugins' && message)) {
+    } else if (endpoint === 'openAI') {
```

Comment:
do you mind fixing getIcon to match getMinimalIcon in structure? we can take this opportunity to clean up the code here